kerastuner/engine/tuner.py

```diff
@@ -310,6 +310,16 @@ def on_epoch_end(self, execution, model, epoch, logs=None):
         self._checkpoint_execution(execution, force=True)
         self._display.on_epoch_end(execution, model, epoch, logs=logs)
 
+        # report intermediate result to the `Oracle`.
+        if self.objective in logs:
+            oracle_response = self.oracle.report_status(
+                execution.trial_id,
+                "RUNNING",
+                score=logs[self.objective],
```

Comment:
Presumably this will need to be modified when we support multi-objective

Reply:
Agreed. We could report all metrics to the Oracle and let it choose which ones it keeps and which one it ignores. Then we don't have to deal with objective configuration in the Tuner at all.